minecraft/protocol.go

```diff
@@ -41,3 +41,7 @@ func (p proto) ConvertToLatest(pk packet.Packet, _ *Conn) []packet.Packet { retu
 func (p proto) ConvertFromLatest(pk packet.Packet, _ *Conn) []packet.Packet {
 	return []packet.Packet{pk}
 }
+
+// DefaultProtocol is the Protocol implementation using as default, In default it is current protocol, version and packet
+// pool and does not convert any packets, as they are already of the right type.
+var DefaultProtocol Protocol = proto{}
```

Comment:
DefaultProtocol can be assigned to now, which I don't think is what we'd want. I'd just make this `var DefaultProtocol proto`, that way it can be used but not assigned to.

Reply:
ok